v2/css/main.css

```diff
@@ -327,6 +327,11 @@ header[role="header"] form {
   padding-bottom: 10px;
   text-align: right;
 }
+header[role="header"] .site-name img{
```

Comment:
BTW, main.css is generated from main.less. So we should really edit main.less instead.